src/trs/lexicon.rs

```diff
@@ -35,11 +35,11 @@ pub struct GeneticParams {
 }
 
 pub trait LexDisplay {
```

Comment:
Because `Operator` is now `Display`, I don't think we need `LexDisplay` at all. Can you remove it?